fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -405,6 +414,58 @@ public CommandProcessingResult withdrawal(final Long savingsId, final JsonComman
                 .build();
     }
 
+    @Transactional
+    @Override
+    public CommandProcessingResult withdrawToLinkedAccount(final Long savingsId, final JsonCommand command) {
+
```

Comment:
It seems majorly a copy-paste of existing functionality, i kindly asking you to extract the common logic into methods and reuse them.